### ISSUE TITLE
Video

### ISSUE THREAD

**Issue by JasmineCT** [JasmineCT/portfolio#16]
I'm trying to upload a video banner but when I export it with Media Encoder it takes away the transparent bounds. So when i upload the video it shows the black boarders. How can I take that away.
@thomasjbradley

**Comment by thomasjbradley** [comment]
You can’t. Videos on the web cannot be transparent—as far as I know.

**Comment by JasmineCT** [comment]
Okay, is there a way when i use either the embed container to not show those parts?

**Comment by thomasjbradley** [comment]
Yeah, probably. You’d have to come up with your own aspect ratio container to do it though.
